### PR TITLE
feat(schemas): add user verifications column

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add user verifications column, this is a jsonb column, the type is defined in `jsonb-types.ts`.

## Testing

Tested in local database.

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/schemas/tables/users.sql [hunk 1]
```diff
@@ -16,6 +16,7 @@ create table users (
   application_id varchar(21),
   identities jsonb /* @use Identities */ not null default '{}'::jsonb,
   custom_data jsonb /* @use JsonObject */ not null default '{}'::jsonb,
+  verifications jsonb /* @use Verifications */ not null default '[]'::jsonb,
```
Comment (simeng-li) [inline packages/schemas/tables/users.sql:19]: mfa_verifications?

**Review by darcyYe — APPROVED** [review]

**Comment by darcyYe** [comment]
maybe should fix PR title, the change is on schemas and core.

File: packages/schemas/tables/users.sql (hunk 1 above)
Comment (wangsijie) [reply packages/schemas/tables/users.sql]: updated

File: packages/schemas/src/foundations/jsonb-types.ts [hunk 2]
```diff
@@ -186,6 +174,51 @@ export const mfaGuard = z.object({
...
+export type Identities = z.infer<typeof identitiesGuard>;
+
+const baseMfaVerification = {
+  id: z.string(),
+  createdAt: z.date(),
+};
+
+const mfaVerificationGuard = z.discriminatedUnion('type', [
+  z.object({
+    type: z.literal(MfaFactor.TOTP),
```
Comment (xiaoyijun) [inline packages/schemas/src/foundations/jsonb-types.ts:196]: It seems that this `type` should be a part of the `baseMfaVerification` above.

**Review by simeng-li — APPROVED** [review]
LGTM

**Pull request merged**